custom/skills.yaml

```diff
@@ -15,7 +15,7 @@ skills:
         default_configurations:
             reset:             [0.01, 0.0, -1.57, -1.57, 0.0]
             handover:          [0.40, -0.2, 0.0, -1.37, 0.0]
-            handover_pose:     [0.40, -0.2, 0.0, -1.37, 0.0]
+            handover_pose:     [0.40, -1.0, 0.0, -1.57, 0.0]
```

Comment:
The handover_pose appears to only be used in the take_out_the_garbage_challenge. Should we give this another name?

Reply:
Yes the name should be changed, however the name should also be changed in the challenge itself. Merging this pull request should only be done if this has been changed.

Reply:
Oke apparentely this pose is not even used in master and it is now fixed in rwc2019 so it can be merged